tethysext/atcore/services/resource_condor_workflow.py

```diff
@@ -0,0 +1,126 @@
+"""
+********************************************************************************
+* Name: resource_condor_workflow.py
+* Author: gagelarsen
+* Created On: December 11, 2020
+* Copyright: (c) Aquaveo 2020
+********************************************************************************
+"""
+import os
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+from tethys_sdk.jobs import CondorWorkflowJobNode
+
+from tethysext.atcore.models.app_users import Resource
+from tethysext.atcore.job_scripts import update_resource_status as update_resource_status_script
+
+
+class ResourceCondorWorkflow(object):
+    """
+    Helper class that prepares and submits the new project upload jobs and workflow.
+    """
+    def __init__(self, user, workflow_name, workspace_path, resource_db_url, resource_id,
+                 scheduler, job_manager, status_keys, **kwargs):
+        """
+        Constructor.
+
+        Args:
+            user(auth.User): Django user.
+            workflow_name(str): Name of the job.
+            workspace_path(str): Path to workspace to be used by job.
+            resource_db_url(str): SQLAlchemy url to Resource database.
+            resource_id(str): ID of associated resource.
+            scheduler (Scheduler): The condor scheduler for the application
+            job_manager (JobManger): The condor job manager for the application.
+            status_keys (list): A list of resource status keys to be passed to the update status script.
```

Comment:
Pick one format not both: var (type) or var(type)

Reply:
var (type)